aws-sdk-rails.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency('concurrent-ruby', '>= 1.3.1') # Utilities for concurrent processing
   spec.add_dependency('railties', '>= 5.2.0') # encrypted credentials
   spec.add_development_dependency('rails')
+  spec.add_development_dependency('rspec-rails', '~> 6.1')
```

Comment:
Should we always pull the latest here?

Reply:
It's up to you - I have not used rspec-rails. Would major versions break the code here? I assume if it does it would be simple to constrain back.

Reply:
actually, let's target it since clearly it doesn't work for all rails versions

Reply:
though, we should probably target rails as well

Reply:
https://github.com/rspec/rspec-rails/blob/v6.1.3/Changelog.md#600--2022-10-10 on breaking changes for ex